composer.json

```diff
@@ -4,12 +4,23 @@
     "license": "GPLv2",
     "config": {
         "optimize-autoloader": true,
-        "classmap-authoritative": true
+        "classmap-authoritative": true,
+        "platform": {
+            "php": "7.0.8"
+        }
     },
     "replace": {
         "guzzlehttp/guzzle": "*"
     },
     "require": {
         "aws/aws-sdk-php": "^3.93"
+    },
+    "require-dev": {
```

Comment:
This is the "standard" way that we do the `vendor-bin` tools, like in other apps.